client/me/purchases/manage-purchase/index.jsx

```diff
@@ -156,10 +156,45 @@ const ManagePurchase = React.createClass( {
 		);
 	},
 
+	getExpiringText( purchase ) {
+		if ( purchase.expiryStatus === 'manualRenew' ) {
+			return this.translate( '%(purchaseName)s will expire and be removed from your site in %(expiry)s. ' +
+				'Please, add a credit card if you want it to autorenew. ',
+				{
+					args: {
+						purchaseName: getName( purchase ),
+						expiry: this.moment( purchase.expiryMoment ).fromNow()
+					}
+				}
+			);
+		}
+		if ( isMonthly( purchase.productSlug ) ) {
+			const expiryMoment = this.moment( purchase.expiryMoment );
+			const daysToExpiry = this.moment( expiryMoment.diff( this.moment() ) ).format( 'D' );
+
+			return this.translate( '%(purchaseName)s will expire and be removed from your site in %(expiry)s days. ',
+				{
+					args: {
+						purchaseName: getName( purchase ),
+						expiry: daysToExpiry
+					}
+				}
+			);
+		}
+
+		return this.translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s.',
```

Comment:
hmmm, that would be neat, but since the params are different, I can't figure out a way to don't repeat the texts without overcomplicating the logic of the method :/